/**:
  ros__parameters:
    # Publishers ============================
    FOUR_IN_ONE_TOPIC:    "/EL/four_in_one"
    NPK_TOPIC:            "/EL/npk"
    VOLTAGE_TOPIC:        "/EL/voltage"
    DRILL_MASS_TOPIC:     "/EL/drill/mass"
    CONTAINER_MASS_TOPIC: "/EL/container/mass"
    IMU_TOPIC:            "/EL/imu_data" # avionics_interfaces/msg/Imu.msg
    IMU_NAV_TOPIC:        "/EL/imu" # sensor_msgs/Imu.msg
    MAG_TOPIC:            "/EL/mag_data"
    MAG_CAL_TOPIC:        "/EL/mag"
    MAG_RAW_TOPIC:        "/EL/mag_raw"
    POTENTIOMETER_TOPIC:  "/EL/potentiometer"
    SPECTRO_TOPIC:        "/EL/spectro_response"
    LASER_TOPIC:          "/EL/laser_response"
    SERVO_TOPIC:          "/EL/servo_response"
    LED_TOPIC:            "/EL/led_response"
    NODE_STATE_TOPIC:     "/EL/node_state"

    # Config
    MASS_CONFIG_REQ_MCU_TOPIC:  "/EL/mass_config_mcu_req"
    MASS_CONFIG_TOPIC:          "/EL/mass_config_response"

    POT_CONFIG_REQ_MCU_TOPIC:  "/EL/pot_config_mcu_req"
    POT_CONFIG_TOPIC:          "/EL/pot_config_response"

    SERVO_CONFIG_REQ_MCU_TOPIC:  "/EL/servo_config_mcu_req"
    SERVO_CONFIG_TOPIC:          "/EL/servo_config_response"

    ACCEL_CONFIG_REQ_MCU_TOPIC:  "/EL/accel_config_mcu_req"
    ACCEL_CONFIG_TOPIC:          "/EL/accel_config_response"

    GYRO_CONFIG_REQ_MCU_TOPIC:  "/EL/gyro_config_mcu_req"
    GYRO_CONFIG_TOPIC:          "/EL/gyro_config_response"

    MAG_CONFIG_REQ_MCU_TOPIC:  "/EL/mag_config_mcu_req"
    MAG_CONFIG_TOPIC:          "/EL/mag_config_response"

    # Subscribers ===========================
    SPECTRO_REQ_TOPIC:    "/EL/spectro_req"
    SERVO_REQ_TOPIC:      "/EL/servo_req"
    LASER_REQ_TOPIC:      "/EL/laser_req"
    LED_REQ_TOPIC:        "/EL/LedCommands"

    # Config
    MASS_CONFIG_REQ_JETSON_TOPIC:   "/EL/mass_config_req"
    POT_CONFIG_REQ_JETSON_TOPIC:    "/EL/pot_config_req"
    SERVO_CONFIG_REQ_JETSON_TOPIC:  "/EL/servo_config_req"
    ACCEL_CONFIG_REQ_JETSON_TOPIC:  "/EL/accel_config_req"
    GYRO_CONFIG_REQ_JETSON_TOPIC:   "/EL/gyro_config_req"
    MAG_CONFIG_REQ_JETSON_TOPIC:    "/EL/mag_config_req"

    # Calibration
    CONTAINER_MASS_CALIB_OFFSET_TOPIC:  "/EL/container/mass_calib_offset"
    DRILL_MASS_CALIB_OFFSET_TOPIC:      "/EL/drill/mass_calib_offset"

    CONTAINER_MASS_CALIB_SCALE_TOPIC:   "/EL/container/mass_calib_scale"
    DRILL_MASS_CALIB_SCALE_TOPIC:       "/EL/drill/mass_calib_scale"

    IMU_CALIB_TOPIC:                    "/EL/imu_calib"